CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+### enhancement 
+- allow separation of resource settings on KSM and forwarder
```

Comment:
You need to follow the changelog convention

Reply:
i am: https://github.com/newrelic/release-toolkit/blob/main/README.md#validate-markdown 

once its merged, it creates the fancy header w/ the emojis and things

Reply:
https://github.com/newrelic/release-toolkit/blob/main/README.md#render-markdown-and-update-markdown 
in the step render-markdown

Reply:
I was told it had to be like this: [PR Guide](https://github.com/newrelic/nri-kubernetes/blob/main/CONTRIBUTING.md#pull-requests)

Would need to be like this: 

```
### enhancement 
- Allow separation of resource settings on KSM and forwarder @jddcarreira [#1130](https://github.com/newrelic/nri-kubernetes/pull/1130)
```

Reply:
oh i see what you're saying 
ok will update